timemachine/cpp/src/kernels/k_rotations.cuh

```diff
@@ -25,13 +25,13 @@ void __global__ k_rotate_coordinates(
 // at the translation as the final step, if SCALE=true then the translation that is provided will be scaled by the box vectors.
 template <typename RealType, bool SCALE>
 void __global__ k_rotate_and_translate_mols(
-    const int num_samples,
+    const int num_batches,
```

Comment:
Does this process *multiple* batches? Or do you mean `batch_size` here?

Reply:
Renamed `batch_size` in https://github.com/proteneer/timemachine/pull/1241/commits/7fb988bf6bab6af0288abfefbac72066aa529f74